docs/release-notes/cloud/cloud-2023-10.md

```diff
@@ -30,6 +30,13 @@ These release notes are for the Codacy Cloud updates during October 2023.
 -   TSQLLint can now also detect and analyze files with an `sql` extension. (TS-543)
 -   Codacy no longer supports and now skips the analysis of branches named `HEAD` or matching the pattern `refs/heads/*`, as these are Git reserved terms. (IO-830)
 
+## Feature removal
+
+Due to their limited adoption, we removed the following options from the menu of each issue listed on the **Issues** page:
+
+-   GitHub: Create pull request comments and GitHub issues.
+-   Bitbucket: Create pull requests comments and Jira issues. (CY-6535)
```

Comment:
```suggestion
-   Bitbucket: Create pull requests comments and Bitbucket issues.
-   Jira: Create Jira issues. (CY-6535)
```

Reply:
@mrfyda This is not consistent with the [previous version of the documentation](https://github.com/codacy/docs/pull/1836/files#diff-e8e58aa758ed6911901c2b230d8f5c9ea696208c3c86317044d9ae6529f6a552L9)... We were not mentioning **Bitbucket issues**... 🤔 

For Bitbucket integration we had:
`When the integration is enabled, you can also create pull request comments and Jira issues directly from Codacy when browsing the existing issues on the repository.`